templates/c2s-pharmacy-finder/1/rancher-compose.yml

```diff
@@ -2,7 +2,8 @@ version: '2'
 
 catalog:
   name: nhsuk_c2s

```

Comment:
Can you remove the ^M - https://github.com/nhsuk/nhsuk-rancher-templates/pull/13/commits/1f97e9d75579968536bca8dec240ae322c74cd70